build.gradle

```diff
@@ -1013,8 +1013,19 @@ project('test:system') {
         commandLine './kubernetes/setupTestPod.sh'
     }
 
+    task runFluentBitUninstall(type: Exec) {
+        environment 'KEEP_PVC', System.getProperty("keepSystemTestLogs", "")
+        commandLine './kubernetes/fluentBitSetup.sh', 'uninstall'
+    }
+
+    task runFluentBitInstall(type: Exec) {
```

Comment:
What about if the cluster already has a fluentbit service running? Is this going to install/uninstall it on the default namespace anyway?

Reply:
By default it will only uninstall a helm installation with the name `pravega-fluent-bit` on the default namespace yes, so there should be a low probability of conflicts with other fluent-bit deployments unrelated to pravega. However I'll remove it anyways as other components do not follow this pattern.